web/src/routes/(user)/people/[personId]/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -81,7 +81,7 @@
   $: thumbnailData = getPeopleThumbnailUrl(person);
   $: if (person) {
     handlePromiseError(updateAssetCount());
-    handlePromiseError(assetStore.updateOptions({ personId: person.id }));
+    handlePromiseError(assetStore.updateOptions({ personId: person.id, isArchived: false }));
```

Comment:
Let's extract the common options up to a new variable so they will always be in sync.